wurst/systems/core/Experience.wurst

```diff
@@ -96,7 +96,7 @@ init
                 let pos = troll.getPos()
                 let killer = ENUM_GROUP
                     ..enumUnitsInRange(pos, expRange, Filter(() -> begin
-                        return GetFilterUnit().getOwner().getId() < bj_MAX_PLAYER_SLOTS
```

Comment:
How did the previous logic match neutral units? Aren't `NEUTRAL_PASSIVE` and `NEUTRAL_HOSTILE` both at least `bj_MAX_PLAYER_SLOTS`?

Reply:
bj_MAX_PLAYER_SLOTS returned 28 in my tests, NEUTRAL_AGGRESSIVE is index 27 I believe.

Reply:
Ah, thought it would return 24.